### PR TITLE
Open Sourced Planner connector

### PR DESCRIPTION
**Pull request by gijeyaku**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### CODE REVIEW

File: connectors/Planner/apiProperties.json [hunk 1]
```diff
@@ -0,0 +1,410 @@
...
+          ],
+          "x-ms-apimTemplateParameter.propertyParentPath": "@body()",
+          "x-ms-apimTemplateParameter.propertySubPath": "assignments",
+          "x-ms-apimTemplateParameter.newPropertyPath": "@body()._assignments",
+          "x-ms-apimTemplateParameter.keyName": "userId",
+          "x-ms-apimTemplateParameter.valueName": "value",
+          "x-ms-apimTemplate-policySection": "Response"
+        },
+        "templateId": "ConvertObjectToArray",
+        "title": "Convert object of assigness to array for a task"
```
Comment (JatinSanghvi) [inline connectors/Planner/apiProperties.json]: Minor comment: Should this be `assignees` or `assignments`?

File: connectors/Planner/Readme.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+
+## Planner Connector
+Planner API in Microsoft Graph lets you create tasks and assign them to users in a group in Office 365. It enables you to easily bring together teams, tasks, documents, and conversations for better results. This connector exposes a subset of this API as operations in Microsoft Power Automate and PowerApps.
```
Comment (JatinSanghvi) [inline connectors/Planner/Readme.md]: `PowerApps` is `Power Apps` now.

Comment (gijeyaku) [reply connectors/Planner/Readme.md]: Fixed. Thanks.

File: connectors/Planner/apiProperties.json (hunk 1 above)
Comment (gijeyaku) [reply connectors/Planner/apiProperties.json]: Fixed. Thanks.

File: connectors/Planner/Readme.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+```paconn
+paconn create --api-def apiDefinition.swagger.json --api-prop apiProperties.json --secret <client_secret>
+```
+
+## Supported Operations
+The connector supports the following operations:
+* `Get a task`: Get an existing Planner task
+* `Create a task`: Create a new task in Planner
+* `Update a task (V2)`: Update an existing Planner task
```
Comment (JatinSanghvi) [inline connectors/Planner/Readme.md]: Any reason this is `Update a task (V2)` and not `Update a task`?

File: connectors/Planner/Readme.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+* `Add assignees to a task`: Add assignees an existing Planner task
+* `Unassign users from a task`: Unassign users from an existing Planner task
```
Comment (JatinSanghvi) [inline connectors/Planner/Readme.md]: Can we have either:
```
* `Add assignees to a task`: Add assignees to an existing Planner task
* `Remove assignees from a task`: Remove assignees from an existing Planner task
```

or 

```
* `Assign users to a task`: Assign users to an existing Planner task
* `Unassign users from a task`: Unassign users from an existing Planner task
```

File: connectors/Planner/Readme.md (hunk 3 above)
Comment (gijeyaku) [reply connectors/Planner/Readme.md]: Only the (V2) version of the operation could be open-sourced.

File: connectors/Planner/Readme.md (hunk 4 above)
Comment (gijeyaku) [reply connectors/Planner/Readme.md]: Makes sense. Went with the first one and made it symmetric.

**Pull request merged**